geoengine/workflow.py

```diff
@@ -109,6 +117,37 @@ def get_dataframe(self, bbox: QueryRectangle) -> gpd.GeoDataFrame:
 
         debug(f'WFS URL:\n{wfs_url}')
 
+        return wfs_url
+
+    def get_wfs_get_map_curl(self, bbox: QueryRectangle) -> str:
```

Comment:
rather `get_wfs_get_feature_curl` because thats the name of the WFS operation

Reply:
aaf9b22